pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "execexam"
-version = "0.3.1"
+version = "0.3.2"
 description = "ExecExam runs executable examinations, providing feedback and assistance!"
-authors = ["Gregory M. Kapfhammer <gkapfham@allegheny.edu>"]
+authors = ["Hemani Alaparthi <alaparthi01@allegheny.edu>","Gregory M. Kapfhammer <gkapfham@allegheny.edu>"]
```

Comment:
@gkapfham Are we adding everyone who has contributed to `execexam` thus far into the authors section of the `pyproject.toml` file? Is this something we should be adding to our PRs in the future?

Reply:
Hi @PCain02 yes, everyone who contributes to the PR should be listed in `authors`.